package.json

```diff
@@ -11,20 +11,22 @@
         "lint:js-fix": "eslint --ext .ts --fix ./src/*",
         "lint:types": "tsc --noEmit",
         "test:trafficlight": "rm -rf ./bin && tsc && node bin/src/index.js",
-        "docker-build": "rm -rf ./bin && tsc",
-        "docker-run": "node bin/src/index.js"
```

Comment:
These didn't really seem to have anything to do with docker and were just building / starting the project.

Reply:
We should also make test:trafficlight just call yarn run build; yarn run start (which is where the split came from, so docker could pre-build)